src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
 public class WorkWithFile {
+    private ArrayList<DataFromFile> dataList = new ArrayList<>();
 
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile work = new WorkWithFile();
+        work.getDataFile(fromFileName);
+        work.writeToFile(toFileName, work.calculating());
+    }
+
+    private void getDataFile(String fromFileName) {
+        try (FileInputStream filein = new FileInputStream(fromFileName)) {
+            if (filein.available() == 0) {
+                throw new RuntimeException("File is empty");
+            } else {
+                BufferedReader br = new BufferedReader(new InputStreamReader(filein));
```

Comment:
just use try with resources and `BufferedReader`, method `readLine()` for example
no need to overcomplicate this and use things like input streams